prebid-server/endpoints/openrtb2/pbs-endpoint-auction.md

```diff
@@ -158,16 +160,11 @@ Prebid Server reads the OpenRTB privacy fields:
 - user.ext.consent
 - device.lmt
 
-#### Other OpenRTB Fields
-
-Prebid Server doesn't do any special processing on any other fields, but passes them
-all to the bid and analytics adapters.
```

Comment:
Worded differently in the section header. Please advise if you disagree this is an improvement.